### PR TITLE
Test

### PR DESCRIPTION
**Pull request by HourglassMagi**
Added 3 Jest tests for database

### CODE REVIEW

**Comment by HourglassMagi** [comment]
[PR Laurent basic Jest tests for database.](https://trello.com/c/v5Fv1ttd/45-pr-laurent-basic-jest-tests-for-database)

File: tests/db.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const db = require('../database/index.js')
+
+describe('Database', () =>{
+
```
Comment (careyl96) [inline tests/db.test.js]: remove space

File: tests/db.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  it('should retrive all records in the database', (done) => {
```
Comment (careyl96) [inline tests/db.test.js]: retrieve*

File: tests/db.test.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    db.client.query(`SELECT * from recipes`, (err, res) => {
+      if(err) {
+        throw err
```
Comment (careyl96) [inline tests/db.test.js]: missing semicolon

File: tests/db.test.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      } else {
+        var results = res.rows.length;
+        console.log(results);
+        expect(results).toBe(101);
+        done();
+      }
+    });
+  });
```
Comment (careyl96) [inline tests/db.test.js:16]: insert space after this line

File: tests/db.test.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      if(err) {
+        throw err
+      } else {
+        var record = res.rows[0].id;
+        console.log(record);
+        expect(record).toBe(9);
+        done();
+      }
+    });
+  })
```
Comment (careyl96) [inline tests/db.test.js]: missing semicolon and insert space after this line

File: tests/db.test.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  it('should return undefined if the record id does not exist', (done) => {
+    db.client.query(`SELECT * from recipes where id = 200`, (err, res) => {
+      if(err) {
+        var error = err
```
Comment (careyl96) [inline tests/db.test.js]: I don't think this is right here

**Review by careyl96 — COMMENTED** [review]
fix commented locations

**Review by careyl96 — CHANGES_REQUESTED** [review]
fix commented locations

File: tests/db.test.js (hunk 6 above)
Comment (HourglassMagi) [reply tests/db.test.js]: @CareyL96 Made the changes you request, awaiting your review. Thanks for your diligence. Much appreciated.

**Review by careyl96 — DISMISSED** [review]
looks good! On line 3 I previously made a mistake on my comment. I mean to say add a space after the arrow which you did, but the space after the comma should still be there. Highly recommend using the airbnb linter though, even if it may be pretty strict.

**Pull request merged**